radio/src/audio.cpp

```diff
@@ -378,14 +377,14 @@ void audioTask(void * pdata)
     DEBUG_TIMER_START(debugTimerAudioDuration);
     audioQueue.wakeup();
     DEBUG_TIMER_STOP(debugTimerAudioDuration);
-    RTOS_WAIT_MS(4);
+    RTOS_WAIT_MS(4); // ???
   }
 }
 #endif
 
 inline void mixSample(audio_data_t * result, int sample, unsigned int fade)
 {
-  *result = limit(AUDIO_DATA_MIN, *result + ((sample >> fade) >> (16-AUDIO_BITS_PER_SAMPLE)), AUDIO_DATA_MAX);
+  *result += (sample >> fade);
```

Comment:
this is IMHO no correct
Audio mixing should be done with saturation, just adding the data might lead to unwanted results

Reply:
indeed!